sparse_strips/vello_hybrid/src/schedule.rs

```diff
@@ -546,9 +556,12 @@ impl Scheduler {
 
                 match scene.encoded_paints.get(paint_id) {
                     Some(EncodedPaint::Image(encoded_image)) => match &encoded_image.source {
-                        ImageSource::OpaqueId(_) => {
+                        ImageSource::OpaqueId(image_id) => {
                             let paint_type = 2_u32;
-                            let paint_packed = (paint_type << 30) | (paint_tex_id & 0x3FFFFFFF);
+                            let skip_paint = !image_cache.contains(*image_id) as u32;
+                            let paint_packed = (paint_type << 30)
+                                | (skip_paint << 29)
```

Comment:
I think this means that we will render something the consumer doesn't expect silently. I suspect at this point we know the consumer hasn't called our APIs correctly, but we're silently rendering their commands anyway. I think we might want to propagate an error or panic instead.